src/DefaultBuilder/samples/SampleApp/Program.cs

```diff
@@ -14,17 +15,14 @@ namespace SampleApp
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
-        }
+            await using var webApp = WebApplication.Create(args);
 
-        public static IHostBuilder CreateHostBuilder(string[] args) =>
```

Comment:
Did we figure out how to fix tooling like EF migrations that rely on the CreateHostBuilder pattern?

Reply:
No that's out of scope for preview4

Reply:
Is it tracked anywhere?

Reply:
@shirhatti ?

Reply:
https://github.com/dotnet/aspnetcore/issues/31861